### PR TITLE
Piping feature

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
Piping can detect changes and hot reload(not browser) without having to restart the server.

Please have a run and let me know your thoughts

### CODE REVIEW

**Comment by jon-shipley** [comment]
It looks like `piping` will still be running even in `production`.  If so, I'd suggest not doing that - this should be a developer convenience tool.

**Comment by GeorgeChatzigiannis** [comment]
@jon-shipley made changes to prevent it from running on production

**Comment by pris54** [comment]
@GeorgeChatzigiannis will it also detect SASS changes?

**Comment by GeorgeChatzigiannis** [comment]
@pris54 It does detect SASS changes but we also need to recompile once there is a change.

File: admin/app.js [hunk 1]
```diff
@@ -49,8 +49,8 @@ const index = require('./routes/index')
 const testDeveloper = require('./routes/test-developer')
 const admin = require('./routes/admin')
 
-let app
-if (piping()) app = express()
+if (process.env.NODE_ENV !== 'production') piping()
```
Comment (jon-shipley) [inline admin/app.js]: As per our offline review (@GuyHarwood, @GeorgeChatzigiannis)  - please can you change this check to look for 'development' in order to run `piping`

**Pull request merged**